packages/node/src/node.ts

```diff
@@ -1 +1,77 @@
-export default class Node {}
+import * as cf from "@counterfactual/cf.js";
+import machine from "@counterfactual/machine";
+import * as ethers from "ethers";
+import { EventEmitter } from "eventemitter3";
+import firebase from "firebase";
+
+export class Node extends EventEmitter {
```

Comment:
@cf19drofxots Instead of extending from `EventEmitter`, maybe it's best to delegate to a private instance `this.eventEmitter`, so we can white-list events listened via `on`.